mypy_primer.py

```diff
@@ -964,6 +1035,11 @@ def inner() -> Optional[int]:
         location="https://github.com/psf/black",
         mypy_cmd="{mypy} src",
         pip_cmd="{pip} install types-dataclasses types-typed-ast",
+        mypyc_cmd=(
+            "{python} setup.py --use-mypyc install "
+            "&& {python} -m pip install pytest"
+            "&& {python} -m pytest -k 'not incompatible_with_mypyc' tests/test_black.py"
```

Comment:
Instead of passing a bespoke set of flags here, mypy should configure an ini option in `conftest.py` so you can run
```suggestion
            "&& {bin}/pytest --mypy-primer
```

Reply:
I'm not convinced the right idea is to run Black's own test suite here, maybe better to run `{python} -m black check .`